kythe/go/extractors/cmd/bazel/bazel_go_extractor/bazel_go_extractor.go

```diff
@@ -32,16 +31,12 @@ import (
 	"kythe.io/kythe/go/extractors/govname"
 	"kythe.io/kythe/go/util/vnameutil"
 
-	"bitbucket.org/creachadair/shell"
-	"bitbucket.org/creachadair/stringset"
-
 	apb "kythe.io/kythe/proto/analysis_go_proto"
 	gopb "kythe.io/kythe/proto/go_go_proto"
-	spb "kythe.io/kythe/proto/storage_go_proto"
 )
 
 var (
-	corpus = flag.String("corpus", "kythe", "The corpus label to assign (required)")
+	corpus = flag.String("corpus", "kythe.io", "The corpus label to assign (required)")
```

Comment:
The other extractors use "kythe" as a default.  Can we keep them consistent?

Reply:
✔︎